src/routes/disclaimer/containers/DisclaimerPage.js

```diff
@@ -44,6 +45,7 @@ const mapDispatchToProps = dispatch => ({
 })
 
 export default compose(
+  withLocationLayout(null),
```

Comment:
What is `null` in this case? Couldn't get it easily by looking at the implementation.

Reply:
Ok, got it now :P